keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -550,8 +578,8 @@ type ServiceImport struct {
 type ServiceImportType string
 
 const (
-  // ClusterSetIP are only accessible via the ClusterSet IP.
-  ClusterSetIP ServiceImportType = "ClusterSetIP"
+  // MultiClusterIP are only accessible via the Multicluster IP.
```

Comment:
Let's just call this specifically what it is - the IP of the multicluster service